### PR TITLE
RDF improvements

### PR DESCRIPTION
**Pull request by hornc**
**What does this PR achieve?**
* Restores the published Open Library concept URIs to `http://openlibrary.org/<key>` format, as linked to by Wikidata and possibly other sources
* Links (via `owl:sameAs`) VIAF, ISNI, and Wikidata URIs for Authors
  *  [VIAF](https://lod-cloud.net/dataset/viaf) and [Wikidata](https://lod-cloud.net/dataset/wikidata) are part of the [LOD Cloud](https://lod-cloud.net/), and [Open Library](https://lod-cloud.net/dataset/open-library) is not yet properly linked to those datasets even though we have the data to do so.

### Technical

Open Library's original URIs were published as `http://`  as picked up in Wikidata: https://www.wikidata.org/wiki/Property:P648#P1921

As a URI I beleive it makes more sense to keep the name consistent and use redirects to enable https / TLS 

I can't find a definitive argument for absolutely keeping the `http://` prefix , but this supports it https://www.w3.org/DesignIssues/Security-NotTheS.html

These URI strings serve primarily as _identifiers_  (_Concept URIs_) rather_ than client / server TLS protocol resolvers (_Data URLs_). I wish there was a good treatment of this issue somewhere: http vs. https in URIs.

The best I have found suggest it is best to continue using whatever your URIs started as, and changing them adds no value and likely breaks things. 

### Testing

Author RDF with `owl:sameAs`:

```
<?xml version="1.0" encoding="utf-8"?>
<rdf:RDF
  xmlns:rdf='http://www.w3.org/1999/02/22-rdf-syntax-ns#'
  xmlns:rdfs='http://www.w3.org/2000/01/rdf-schema#'
  xmlns:bio='http://purl.org/vocab/bio/0.1/'
  xmlns:rdg2='http://rdvocab.info/ElementsGr2/'
  xmlns:dcterms='http://purl.org/dc/terms/'
  xmlns:foaf='http://xmlns.com/foaf/0.1/'
  xmlns:ov='http://open.vocab.org/terms/'
  xmlns:owl='http://www.w3.org/2002/07/owl#'
>
    

    <foaf:Person rdf:about="http://openlibrary.org/authors/OL6848355A">
            <foaf:name>Homer</foaf:name>
    

            <rdg2:variantNameForThePerson>Homère</rdg2:variantNameForThePerson>
    <rdg2:variantNameForThePerson>Homère.</rdg2:variantNameForThePerson>
    <rdg2:variantNameForThePerson>Homere/</rdg2:variantNameForThePerson>
    <rdg2:variantNameForThePerson>Homerus</rdg2:variantNameForThePerson>
    <rdg2:variantNameForThePerson>Homerus ca. 8th-7th cent. B.C.</rdg2:variantNameForThePerson>
    <rdg2:variantNameForThePerson>Homeros</rdg2:variantNameForThePerson>
    <rdg2:variantNameForThePerson>Omero</rdg2:variantNameForThePerson>

        
            <rdg2:titleOfThePerson>ca. 8th-7th cent. B.C.</rdg2:titleOfThePerson>
    


      <owl:sameAs rdf:resource="http://isni.org/isni/000000036354151X"/>
      <owl:sameAs rdf:resource="http://www.wikidata.org/entity/Q6691"/>
      <owl:sameAs rdf:resource="http://viaf.org/viaf/224924963"/>
      <foaf:page>
          <foaf:Document rdf:about="http://en.wikipedia.org/wiki/Homer">
              <rdfs:label>Wikipedia entry</rdfs:label>
          </foaf:Document>
      </foaf:page>
      
      <foaf:page>
          <foaf:Document rdf:about="http://www.online-literature.com/homer/">
              <rdfs:label>Biography &amp; Works</rdfs:label>
          </foaf:Document>
      </foaf:page>
      
      <foaf:depiction>
          <foaf:Image rdf:about="https://covers.openlibrary.org/b/id/6868139-M.jpg" />
      </foaf:depiction>
    </foaf:Person>

    <!-- administrative -->
    <rdf:Description rdf:about="">
        <dcterms:modified rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-04-06T02:11:20.140371</dcterms:modified>
        <dcterms:created rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2013-03-28T07:50:47.897206</dcterms:created>
        <ov:versionnumber>3</ov:versionnumber>
    </rdf:Description>
</rdf:RDF>
```
Validated on https://www.w3.org/RDF/Validator/

### CODE REVIEW

**Comment by cclauss** [comment]
Can you help me to understand why we are preferring insecure connections over secure ones?

**Comment by hornc** [comment]
I was hoping to sort out minor RDF issues and include other linked data identifiers or authors, editions, and works from our data model. Specifically [VIAF](https://lod-cloud.net/dataset/viaf), ISNI, [Wikidata](https://lod-cloud.net/dataset/wikidata), and possibly [DNB](https://lod-cloud.net/dataset/dnb-gemeinsame-normdatei) -- although this will needs some additional work for importing and linking identifiers.

Some pages include schema.org structured data, it'd be nice if we had a unified presentation of all our links with other sources too.

OL's author RDF uses the now deprecated http://metadataregistry.org/schemaprop/show/id/1131.html The entire http://rdvocab.info/ElementsGr2 schema namespace is deprecated according to http://metadataregistry.org/schema/show/id/15.html  Not sure what a current equivalent is. DNB uses their own definition https://d-nb.info/standards/elementset/gnd#variantNameForThePerson

**Comment by hornc** [comment]
For RDF, `owl:sameAs` looks like the appropriate way to indicate equivalence for VIAF, ISNI, and Wikidata etc.

**Comment by hornc** [comment]
I should review #130 in the context of these change if I'm trying to reboot the usefulness of Open Library's RDF.

**Comment by cdrini** [comment]
(Heads up schema.org will be updated to use json-ld once #4378 goes through)

**Review by mekarpeles — COMMENTED** [review]
lgtm, needs tested

**Comment by hornc** [comment]
thanks @mekarpeles , I made some more small changes  as http://staging.openlibrary.org/authors/OL112912A.rdf  had a broken depiction photo, and http://staging.openlibrary.org/authors/OL2793649A.rdf showed as a Person when it is an Organization.

File: openlibrary/core/models.py [hunk 1]
```diff
@@ -593,6 +594,15 @@ def __repr__(self):
         return "<Author: %s>" % repr(self.key)
     __str__ = __repr__
 
+
+    def foaf_agent(self):
+        """ FOAF Agent type. http://xmlns.com/foaf/spec/#term_Agent """
```
Comment (cclauss) [inline openlibrary/core/models.py:599]:
```suggestion
        """
        Friend of a friend ontology Agent type. http://xmlns.com/foaf/spec/#term_Agent
        https://en.wikipedia.org/wiki/FOAF_(ontology)
        """
```

**Review by hornc — COMMENTED** [review]
@mekarpeles tested on staging, and locally. I think this is ready.

**Pull request merged**